README.md

```diff
@@ -1,36 +1,47 @@
 # Bank Transaction Email Reader
-The intention is of this project is to read emails comming from a bank regarding information of a transaction.
 
-The ouput right now is an csv file delimited by pipes ("|") but the goal is to return a json with all the information so you can be able to use this on your mobile, desktop or web applications.
+The intention of this project is to read emails coming from a bank regarding information of a transaction.
 
-> Output Example:
+The output right now is an csv file delimited by pipes ("|") but the goal is to return a json with all the information so you can be able to use this on your mobile, desktop or web applications.
+
+Output Example:
 
     date|currency|amount|merchant|status|type|
     21/12/19 23:40|RD|91.12|None|Reversada|Compra
     21/12/19 23:40|RD|91.12|UBR* PENDING.UBER.COM|Aprobada|Compra
     21/12/19 20:32|RD|125.53|UBR* PENDING.UBER.COM|Aprobada|Compra
 
+# Task Legend
+
+:construction: Work in progress
+
+:white_check_mark: When task is done
+
+:alarm_clock: Pending
 
 ## The current supported banks are:
-  - Banco BHD León (Dominican Republic)
-  - Banco Vimenca (Dominican Republic) `(in progress)`
+  - Banco BHD León (Dominican Republic) :white_check_mark:
+  - Banco Vimenca (Dominican Republic) :construction:
 
-## The current supported email services are:
-  - Gmail: Using the Google api with readonly permission
+## Known Major Tasks TODO
 
+  1. When running the main file, one parameter should be the user email to determine which mail service to use
+  2. Right now it's only expecting multipart emails (threads) not single messages
+  3. How to handle email credentials?
+  4. How to handle email authentication token?
+  5. Add Outlook Email service
+  6. Add a new Email Service 
+     1. Currently only Gmail is supported. Stay tunned for instructions on how to add a new service. 
```

Comment:
don use specific sequence just `1. ` that way it renders automatically

1. blah
1. blah
   1. sub blah